src/components/Header/Header.styled.jsx

```diff
@@ -0,0 +1,289 @@
+import styled, { css } from "styled-components"
+
+export const fontRegular = css` 
+    font-family: "Montserrat-Regular";
+    font-weight: 500;
+    line-height: 15px;
+    font-size: 12px;
+    letter-spacing: 0.04em;
+`;
+
+export const elementMenu = css` 
+    padding-left: 16px;
+    padding-right: 16px;
+    height: 19px;
+    display: grid;
+    align-items: center;
+    ${fontRegular}
+    &:hover {
+        color: black;
+        font-weight: bold;
+        box-shadow: inset -10px -10px 15px rgba(255, 255, 255, 0.8), 
+        inset 10px 10px 15px rgba(70, 70, 70, 0.12);
+      }
+`;
+
+export const ContainerHeder = styled.div` 
+    position: fixed;
+    height: 64px;
+    top: 0;
+    left: 0;
+    display: flex;
+    padding: 0 20px;
+    justify-content: space-between;
+    align-items: center;
+    background-color: white;
+    box-shadow: 0px 2px 4px rgb(0 0 0 / 20%);
+    width: 100%; 
+    @media (max-width: 270px){
+        min-width: 270px;
+    }
+    @media (max-width: 295px){
+        padding: 0 10px;
+    }
+`;
+
+export const LeftBlock = styled.div` 
+    min-height: 30px;
+    width: min-content;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+`;
+
+export const LogoHeader = styled.div` 
+    width: 103px;
+    min-width: 103px;
+    font-family: 'Montserrat-700';
+    font-size: 16px;
+    letter-spacing: 0.08em;
+    color: #000000;
```

Comment:
It will be good if you use color constants because most colors are reused in other places. You can get some constants and create a new one in assets/themes/colors.js (this file you can find in the project after merging the AuthForm branch into the master branch)

Reply:
Thanks for the advice Nastia) I'll try to do it.